ivy/ivy.xml

```diff
@@ -66,6 +66,7 @@
 		<!-- End of Hadoop Dependencies -->
 
 		<dependency org="org.apache.tika" name="tika-core" rev="1.17" />
+        <dependency org="org.apache.any23" name="apache-any23-core" rev="2.0"/>
```

Comment:
Upgrade this to 2.1 and correct formatting.

Reply:
c380e2d